src/main/java/com/redhat/devtools/lsp4ij/LanguageServerWrapper.java

```diff
@@ -1233,4 +1265,13 @@ URI toUri(@NotNull VirtualFile file) {
         return FileUriSupport.getFileUri(file, getClientFeatures());
     }
 
+    /**
+     * Returns the language server wrapper's modification tracker.
+     *
+     * @return the modification tracker
+     */
+    @NotNull
+    ModificationTracker getModificationTracker() {
```

Comment:
Package-projected access to the modification tracker so that `LangaugeServerAccessor` can provide all modification trackers for all language servers for a given file.